benchmarks/nel/scripts/datasets/mewsli_9.py

```diff
@@ -0,0 +1,32 @@
+""" Dataset class for Mewsli-9 dataset. """
+import csv
+from typing import Tuple, Set, List, Dict
+
+from spacy.tokens import Doc
+
+from datasets.dataset import Dataset
+from datasets.schemas import Entity, Annotation
+
+
+class Mewsli9Dataset(Dataset):
+    """ Mewsli-9 dataset. """
+
+    @property
+    def name(self) -> str:
+        return "mewsli_9"
+
+    def _parse_external_corpus(self, **kwargs) -> Tuple[Dict[str, Entity], Set[str], Dict[str, List[Annotation]]]:
+        entities: Dict[str, Entity] = {}
+        annotations: Dict[str, List[Annotation]] = {}
+
+        with open(self._paths["assets"] / "en" / "mentions.tsv", encoding="utf-8") as file_path:
+            for i, row in enumerate(csv.reader(file_path, delimiter="\t")):
+                if i == 0:
+                    continue
+                assert len(row) == 8
```

Comment:
Is the first row in this CSV a header?

Reply:
Yes. Will use a `DictReader` though and discard the check for the first row.